src/clojure/monger/collection.clj

```diff
@@ -524,6 +525,7 @@
 ;;
 
 (defn- build-aggregation-options
+  ^AggregationOptions
```

Comment:
Yes, now I see that a type hint before the argument vector [is supported](http://clojure.org/reference/java_interop#typehints).